### DISCUSSION TITLE
CSS loading order

### DISCUSSION THREAD

**Discussion by tombrouwer** [nuxt/nuxt#18055] [Questions] [answered]
Hi,

I did setup a new project with Nuxt 3. It is working fine. But I do have one problem. I use a UI kit (naive ui) it looks like it is working. Besides the ui kit I use Tailwind. TailwindCSS comes with some CSS to reset the browser default styles. I did setup a CSS file, load it in de config `css: ["@/assets/css/tailwind.css"],`. The Tailwind CSS file is always the last CSS loaded in the head tag of my html. Is there a way to manipulate the order? I tried to use hooks in the config, putted a console log in it, but do not get any log while building or visit the a page.

**Comment by tombrouwer** [comment, marked as answer]
Sorry, the exact problem is described by naive ui (https://www.naiveui.com/en-US/os-theme/docs/style-position).

**Reply by NisuSan** [reply]
@tombrouwer Link you provided is not valid anymore. Can you put your solition here?

**Reply by andreasschiestl** [reply]
https://www.naiveui.com/en-US/os-theme/docs/style-conflict is a current link to the documentation.
